{
  "ID": "oval:org.debian:def:202025658",
  "Class": "vulnerability",
  "Metadata": {
    "Title": "CVE-2020-25658",
    "AffectedList": [
      {
        "Family": "unix",
        "Platform": "Debian GNU/Linux 9",
        "Product": "python-rsa"
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2020-25658",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25658"
      }
    ],
    "Description": "",
    "Debian": {
      "MoreInfo": ""
    }
  },
  "Criteria": {
    "Operator": "AND",
    "Criterias": [
      {
        "Operator": "OR",
        "Criterias": [
          {
            "Operator": "AND",
            "Criterias": null,
            "Criterions": [
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:2",
                "Comment": "all architecture"
              },
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:22223",
                "Comment": "python-rsa DPKG is earlier than 0"
              }
            ]
          }
        ],
        "Criterions": null
      }
    ],
    "Criterions": [
      {
        "Negate": false,
        "TestRef": "oval:org.debian.oval:tst:1",
        "Comment": "Debian 9 is installed"
      }
    ]
  }
}